monkey/monkey_island/cc/services/remote_run_aws.py

```diff
@@ -131,7 +131,8 @@ def _get_run_monkey_cmd_linux_line(bit_text, island_ip):
         return (
             r"wget --no-check-certificate https://"
             + island_ip
-            + r":5000/api/monkey/download/monkey-linux-"
+            + r":5000/api/agent/download/linux "
+            + r"-O monkey-linux-"
             + bit_text
             + r"; chmod +x monkey-linux-"
             + bit_text
```

Comment:
Do we need bit_text here anymore?

Reply:
I don't think so. I have assumed that this is not the task to remove that..

Reply:
It's removed in the windows function in this file. Might as well remove it here too
.